sdk/src/utils/public/apr-calculation.ts

```diff
@@ -0,0 +1,120 @@
+export type RewardToken = {
+  emissionPerWeek: number;
+  tokenPrice: number;
+};
+
+export type PoolAPRReward = {
+  apr: number;
+  rewards: { [k: string]: number } | null;
+};
+
+/**
+ *
+ * @param emissionPerWeek : reward for weekly emission
+ * @param tokenPrice : price of token in USDT
+ * @param tvl : Liquidity is TVL
+ * @returns : Reward APR for token
+ */
+function calculateTokenPoolRewardAPR(
+  emissionPerWeek: number,
+  tokenPrice: number,
+  tvl: number
+): number {
+  return (emissionPerWeek * tokenPrice * (365 / 7)) / tvl;
```

Comment:
các phép tính toán thì mình nên dùng lib nhen em. E.g. `new Decimal(emissionPerWeek).mul(tokenPrice).mul(new Decimal(365).div(7)).div(tvl)`